bundles/org.openhab.core/src/main/java/org/openhab/core/library/items/NumberItem.java

```diff
@@ -142,8 +142,7 @@ public void send(QuantityType<?> command) {
         if (state instanceof QuantityType<?> quantityType) {
             if (dimension == null) {
                 // QuantityType update to a NumberItem without unit, strip unit
-                DecimalType plainState = new DecimalType(quantityType.toBigDecimal());
-                return plainState;
+                return new DecimalType(quantityType.toBigDecimal());
```

Comment:
The constructor knows how to handle this: 

```java
    public DecimalType(Number value) {
        if (value instanceof QuantityType type) {
            this.value = type.toBigDecimal();
        } else if (value instanceof HSBType type) {
```

```suggestion
                return new DecimalType(quantityType);
```

Reply:
done, found one more in the same file....